src/lib/widget/_utilities/api.utilities.ts

```diff
@@ -38,9 +43,6 @@ export function widgetApiFactory(componentApi: ReturnType<typeof _componentApi>)
     journeyStore.reset();
     oauthStore.reset();
     userStore.reset();
-
-    // Fetch fresh journey step
-    journey().start();
```

Comment:
This wasn't needed, and resulted in a journey rendered that may not be relevant for the user that just logged out.